src/test/java/com/google/devtools/build/lib/query2/cquery/ProtoOutputFormatterCallbackTest.java

```diff
@@ -529,6 +529,29 @@ public void outputOnAliasDep() throws Exception {
         .doesNotContain("//test:my_child");
   }
 
+
+  @Test
+  public void incompatibleTarget() throws Exception {
+    writeFile(
+        "test/BUILD",
+        """
+            constraint_setting(name = "incompatible_setting")
+            constraint_value(
+                name = "incompatible",
+                constraint_setting = ":incompatible_setting",
+            )
+            genrule(
+                name = "my_target",
+                cmd = "touch $@",
+                outs = ["out"],
+                target_compatible_with = [":incompatible"],
+            )""");
+
+    options.transitions = Transitions.LITE;
+    AnalysisProtosV2.CqueryResult cqueryResult =
+        getProtoOutput("deps(//test:my_target)", AnalysisProtosV2.CqueryResult.parser());
```

Comment:
I assume `getProtoOutput` before would trigger the `InterruptedException` that auto-failed this test? Could we, just for clarity, add a line verifying the result isn't null and/or checking the warning message?

Also, what are you distinctly testing here compared to the shell test?

Reply:
This was just a leftover from an earlier attempt at a Java test that ultimately failed. I removed it.